parlai/tasks/mnist/agents.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
```

Comment:
add a description of this task after the license info as a docstring, e.g. 
```
"""Some helpful info
about the task.
"""
```

This will get imported into the documentation automatically.